integrationtests/mocks/core_controller_mock.go

```diff
@@ -0,0 +1,195 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Do we know why these 3 new mock files are generated? Their contents don't seem to be used anywhere :thinking: 

Edit: we should probably update/get rid of [those lines](https://github.com/rancher/fleet/blob/main/integrationtests/cli/apply/apply_online_test.go#L1-L3), which seem to be causing generation of these files without need.